_layouts/page_footer.html

```diff
@@ -3,41 +3,41 @@
+            <span class="icon"><svg height="24" aria-label="Twitter" class="octicon octicon-mark-twitter" viewBox="0 0 16 16" version="1.1" width="24" role="img"><path style="stroke-width:0.76190084" d="M 14.115052,3.3110602 C 14.793144,2.8831766 15.313446,2.2046963 15.557558,1.3964597 14.922743,1.7936425 14.221184,2.0808798 13.472693,2.2363913 12.875439,1.5628604 12.021806,1.1428961 11.077048,1.1428961 c -1.8125622,0 -3.2813546,1.5501024 -3.2813546,3.4617346 0,0.2713891 0.027277,0.5358524 0.083581,0.7884225 C 5.1520574,5.2484445 2.7337994,3.8716744 1.1138076,1.775814 0.83112716,2.2888864 0.66959809,2.8831766 0.66959809,3.5170781 c 0,1.2004663 0.57943931,2.2602855 1.46034151,2.882309 C 1.5918242,6.3815587 1.0856326,6.2240662 0.64236318,5.9665437 v 0.04259 c 0,1.677881 1.13070962,3.0774167 2.63425232,3.3943674 -0.2761052,0.081219 -0.5653532,0.121828 -0.8658774,0.121828 -0.211298,0 -0.4179102,-0.020801 -0.6170025,-0.061409 0.4169731,1.3747739 1.6293858,2.3772069 3.0662471,2.4039489 -1.1241315,0.929062 -2.5394004,1.481745 -4.07674862,1.481745 -0.26483598,0 -0.52685215,-0.01486 -0.78323408,-0.04755 1.4528307,0.981558 3.1780103,1.55504 5.0318447,1.55504 6.0385753,0 9.3386723,-5.2753252 9.3386723,-9.8503417 0,-0.1505592 -0.0018,-0.3001205 -0.0084,-0.4477006 C 15.003586,4.0707561 15.56145,3.4606183 16,2.7662903 15.411203,3.041649 14.778216,3.2278574 14.115133,3.3110571 Z" /></svg></span>
             <span>Twitter</span>
           </a></li>
         </ul>
       </div>
       <div class="g-col-3">
         <h3>Standards</h3>
-        <ul>
+        <ul aria-label="standards">
           <li><a href="https://tc39.es/ecma262/">ECMA-262, ECMAScript</a></li>
           <li><a href="https://tc39.es/ecma402/">ECMA-402, Intl API</a></li>
         </ul>
       </div>
       <div class="g-col-3">
         <h3>Proposals</h3>
-        <ul>
+        <ul aria-label="proposals">
           <li><a href="https://github.com/tc39/proposals/blob/master/stage-0-proposals.md">Stage 0</a></li>
           <li><a href="https://github.com/tc39/proposals/blob/master/stage-1-proposals.md">Stage 1</a></li>
-          <li><a href="https://github.com/tc39/proposals#stage-2">Stage 2</a></li>
-          <li><a href="https://github.com/tc39/proposals#stage-3">Stage 3</a></li>
+          <li><a href="https://github.com/tc39/proposals/blob/master/README.md#stage-2">Stage 2</a></li>
+          <li><a href="https://github.com/tc39/proposals/blob/master/README.md#stage-3">Stage 3</a></li>
```

Comment:
@ljharb, by any chance would you happen to know if it's safe to use `HEAD` for these links here too? Your suggestion in https://github.com/tc39/tc39.github.io/pull/211#discussion_r488294262 seems to imply that, but here's what the [Git glossary](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt) says (emphasis mine). 🤔

<dl>
<dt>branch</dt>
<dd>

A "branch" is an active line of development. The most recent commit on a branch is referred to as the tip of that branch. The tip of the branch is referenced by a branch head, which moves forward as additional development is done on the branch. A single Git repository can track an arbitrary number of branches, but <em>your working tree is associated with just one of them (the "current" or "checked out" branch), and HEAD points to that branch</em>.

Source: https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefbranchabranch

</dd>
<dt>HEAD</dt>
<dd>

The current branch. In more detail: Your working tree is normally derived from the state of the tree referred to by HEAD. HEAD is a reference to one of the heads in your repository, except when using a detached HEAD, in which case it directly references an arbitrary commit.

Source: https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefHEADaHEAD

</dd>
</dl>

Reply:
This may be less than ideal because that would link users to a GitHub repo already at a specific commit (see below).

![image](https://user-images.githubusercontent.com/17770407/95694199-aacb9700-0bfe-11eb-984a-a76a03204c28.png)